packages/dds/matrix/src/matrix.ts

```diff
@@ -101,14 +152,30 @@ export class SharedMatrix<T = any>
 
 	private cells = new SparseArray2D<MatrixItem<T>>(); // Stores cell values.
 	private pending = new SparseArray2D<number>(); // Tracks pending writes.
+	private cellLastWriteTracker = new SparseArray2D<CellLastWriteTrackerItem>(); // Tracks last writes sequence numner in a cell.
+	// Tracks the seq number of Op at which policy switch happens from Last Write Win to First Write Win.
+	private setCellPolicySwitchOpSeqNumber: number;
```

Comment:
I'd suggest changing name of this variable to better reflect what it tracks (It's hard to get it from code that it's about FWW)

Reply:
done. Renamed it to "setCellLwwToFwwPolicySwitchOpSeqNumber"